validators/node-api.md

```diff
@@ -0,0 +1,178 @@
+# Node REST API v1
+
+Each SKALE node has `skale_api` container with Flask webserver that provides REST JSON API to interact with it.  
+SKALE node REST API is available locally on the node and could be accessed on http://localhost:3007.
+
+> ❗️ WARNING: REST API provides an access to private methods such as money transfers, node controls, etc. Make sure that port `3007` is available only to your trusted clients.
+
+> Node REST API v1 available in `skale_api > 2.0.0-develop.0`
+
+## Table of contents
+
+- [Node REST API v1](#node-rest-api-v1)
+  * [Table of contents](#table-of-contents)
+  * [API structure](#api-structure)
+  * [Public/private APIs](#public-private-apis)
+  * [REST JSON API](#rest-json-api)
+    + [Logs](#logs)
+      - [Methods](#methods)
+      - [Usage](#usage)
+    + [Node](#node)
+      - [Methods](#methods-1)
+    + [Usage](#usage-1)
+    + [Health](#health)
+      - [Methods](#methods-2)
+      - [Usage](#usage-2)
+    + [sChains](#schains)
+      - [Methods](#methods-3)
+      - [Usage](#usage-3)
+    + [SSL](#ssl)
+      - [Methods](#methods-4)
+      - [Usage](#usage-4)
+    + [Wallet](#wallet)
+      - [Methods](#methods-5)
+      - [Usage](#usage-5)
+
+## API structure
+
+```bash
+/api/{API_VERSION}/{BLUEPRINT_NAME}/{METHOD_NAME}
+```
+
+## Public/private APIs
+
+Methods that are marked as `ALLOWED` could be safely used by validators directly or by their custom tools.  
+Usage examples are available for public APIs.
+
+Direct use:
+
+- `ALLOWED` - API is safe to use directly by custom tools
+- `NOT ALLOWED` - API shouldn't be used directly
```

Comment:
May be it's better to hide such endpoints. What the point to show them in docs?

Reply:
This separation is nominal tbh. Maybe it's better to rename `NOT ALLOWED` to `NOT RECOMMENDED` or something like this. Cause actually they could be called directly with caution.